docs/_writing-tests/h2responsewriter.md

```diff
@@ -0,0 +1,113 @@
+## Response Writer API ##
+
+##### `write_headers(self, headers, status_code, status_message=None, stream_id=None, last=False):`
+Send a HEADER frame that is tracked by the local state machine.
```

Comment:
I'm not sure these one sentence summaries of the two sentence descriptions are adding much.